api/src/test/java/org/pmiops/workbench/api/ConceptSetsControllerTest.java

```diff
@@ -1063,13 +1063,13 @@ public void updateConceptSetConceptsAddAndRemove() {
     ConceptSetConceptId addedConcept3 =
         new ConceptSetConceptId()
             .conceptId(CRITERIA_CONDITION_3.getConceptId())
-            .standard(CRITERIA_CONDITION_3.getIsStandard());
+            .standard(CRITERIA_CONDITION_3.isIsStandard());
```

Comment:
Most of these should be fixed now. There are a couple remaining of the "isIs"pattern. This is because these are tired to Rawls (api/build/swagger3/org/pmiops/workbench/rawls/model/RawlsWorkspaceDetails.java) and Firecloud (api/build/swagger3/org/pmiops/workbench/firecloud/model/FirecloudWorkspaceDetails.java). I suspect that this is unrelated to this PR.